packages/common/src/downloads/build-existing-exports-portal-query.ts

```diff
@@ -143,11 +201,37 @@ function buildExportTypesClause(
   });
 }
 
-function getSpatialRefTypeKeyword(
-  supportsProjection: boolean,
-  spatialRefId?: string
-) {
-  return `spatialRefId:${
-    supportsProjection && spatialRefId ? spatialRefId : WGS84_WKID
-  }`;
+/**
+ * Generates typekeyword for identifying which spatialRefId an export is
+ * @param spatialRefId
+ * @param supportsProjection
+ * @returns
+ */
+export function getSpatialRefTypeKeyword(spatialRefId: string) {
+  const parsedSpatialReference = parseSpatialRefId(spatialRefId);
+  const serializedSpatialReference = serializeSpatialReference(
+    parsedSpatialReference
+  );
+  return `spatialRefId:${serializedSpatialReference}`;
+}
+
+/**
+ * Returns the keyword identifying exports by the item they originate from
+ * @param itemId
+ * @returns
+ */
+export function getExportItemTypeKeyword(itemId: string) {
+  return `exportItem:${itemId}`;
+}
+
+/**
+ * Returns the keyword identifying exports by the layer they originate from
+ * @param layerId
+ * @returns
+ */
+export function getExportLayerTypeKeyword(layerId?: number | string) {
+  // NOTE - Layer Id's need to be padded with "0" so that /search results are predictable. Searches for typeKeywords:"exportLayer:1" don't work.
+  // See https://github.com/Esri/hub.js/pull/472 for more information.
+  // TODO - use `filter` when Enterprise Sites adds support.
+  return layerId ? `exportLayer:0${layerId}` : `exportLayer:null`;
 }
```

Comment:
These are the functions that generate appropriate typeKeywords for the exported items. Now it's one set of logic that is used in both tagging and querying.